pkg/cluster/util.go

```diff
@@ -295,35 +300,55 @@ func (c *Cluster) waitPodLabelsReady() error {
 			c.OpConfig.PodRoleLabel: string(Replica),
 		}).String(),
 	}
-	pods, err := c.KubeClient.Pods(namespace).List(listOptions)
-	if err != nil {
-		return err
+	podsNumber = 1
+	if !anyReplica {
+		pods, err := c.KubeClient.Pods(namespace).List(listOptions)
+		if err != nil {
+			return err
+		}
+		podsNumber = len(pods.Items)
+		c.logger.Debugf("Waiting for %d pods to become ready", podsNumber)
```

Comment:
probably that line  should mention that the operator waits for all replicas of that PG cluster and then state their number ?

Reply:
For the debugging purposes it's important to know how many replicas we are waiting for. The person examining the output would already know the context from the prior debug messages in the same session (i.e. when we are waiting for all pods they will see it is because of the statefulset.